ui/src/App.js

```diff
@@ -109,6 +109,28 @@ class IntlRoutes extends Component {
               );
             }}>
             </Route>
+
+            <Route path="/:lan/dashboard/:slug" exact render={props => {
```

Comment:
@jdeanquin-dg Could you check if this is ok? I added a dashboard wrapper div to dashboard pages to be able to target the footer styles in the dashboard pages specifically. It works on my end but may be you can take a look just in case.